tests/framework/db/mssql/QueryBuilderTest.php

```
@@ -643,6 +641,8 @@ public function testDropColumnDropsConstraintOnDb(
 
     private function assertDropColumnOnDb(Connection $db, string $table, string $column): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)